nifi-assembly/pom.xml

```diff
@@ -475,6 +475,12 @@ language governing permissions and limitations under the License. -->
             <version>1.9.0-SNAPSHOT</version>
             <type>nar</type>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-elasticsearch-client-service-6x-nar</artifactId>
+            <version>1.8.0-SNAPSHOT</version>
```

Comment:
These should all be updated to 1.11.0-SNAPSHOT, that might be what's causing the non-cached Maven build failures in Travis.